pkgs/top-level/python-packages.nix

```
@@ -1490,6 +1490,9 @@ in {
     inherit (pkgs) sentencepiece pkgconfig;
   };
 
+  tokenizers = disabledIf (!isPy3k)
+    (callPackage ../development/python-modules/tokenizers { });
```

Move disable to within module. See other comments
```suggestion
  tokenizers = callPackage ../development/python-modules/tokenizers { };
```